app/views/grids/_notes.html.erb

```diff
@@ -87,31 +79,18 @@
   <p><i>Activities should include a materials list, costs and a step-by-step guide to construction with photos. Learn what <a href="https://publiclab.org/notes/warren/09-17-2016/what-makes-a-good-activity">makes a good activity here</a>.</i>
   </p>
 <% end %>
-
+  
 <script>
 (function(){
-  $(".<%= className %>-<%= randomSeed %> .show-all a").click(function(event) {
-    var cName = ".<%= className %>-<%= randomSeed %>";
-    var sName = cName + " tr.extra";
-    $(sName).toggleClass("hide");
-    if ($(sName)[0].classList.contains("hide"))
-      var hiding = true;    
-    var myElement = event.target;    
-    if (event.target.classList.contains("caret")) 
-      myElement = event.target.parentElement;
-    if (!hiding) 
-      myElement.innerHTML = 'Show less <b class="caret" style = "transform: rotate(180deg);"></b>';
-    else
-      myElement.innerHTML = 'Show more <b class="caret"></b>';
-  });
   //setupGridSorters(".<%= className %>-<%= randomSeed %>"); Should order the items by time. Commented out because it reorders the items each time show more/less is clicked
 })()
 </script>
+
 <style>
   th {
     position: sticky;
     top: 0;
-    z-index: 5;
+    z-index: 3;
```

Comment:
@Yavnikaa We don't need this change.

Reply:
Updated that.
As per your previous comment, I figured out that I need to change the file `test/unit/node_shared_test.rb` , and replace 5 by 3. But I don't understand how did removing show more bring about these tests failing. Can you help me with this, and am I doing it right as per the error log in the comment?